RadixWallet/Core/DesignSystem/Components/Thumbnails.swift

```diff
@@ -180,8 +181,8 @@ public struct LoadableImage<Placeholder: View>: View {
 		placeholder: () -> Placeholder
 	) {
 		if let url {
-			if url.isVectorImage {
-				loggerGlobal.warning("LoadableImage: Vector images are not supported \(url)")
+			if url.isUnsupportedVectorImage {
+				loggerGlobal.warning("LoadableImage: Vector image is not supported \(url)")
```

Comment:
And I would change the warning here to `LoadableImage: PDF vector images are not supported \(url)`